zen.lp

```diff
@@ -0,0 +1,596 @@
+// Logic for Zenon Modulo
+
+
+set declared "τ"
+set declared "ι"
+set declared "ϵ"
+set declared "⊤"
+set declared "⊥"
+set declared "¬"
+set declared "∀"
+set declared "∃"
+
+symbol prop  : TYPE
+symbol Type  : TYPE
+
+injective symbol ϵ : prop → TYPE
+injective symbol τ : Type → TYPE
+
+symbol arrow : Type → Type → Type
+constant symbol ι  : Type
+
+// Symbols
+
+symbol ⊤ : prop
+symbol ⊥ : prop
+
+symbol ¬ : prop → prop
+symbol {|and|} : prop → prop → prop
+symbol or  : prop → prop → prop
+symbol imp : prop → prop → prop
+symbol eqv : prop → prop → prop
+
+set infix right 5 "⇒" ≔ imp
+set infix right 10 "∧" ≔ {|and|}
+set infix right 9 "∨" ≔ or
+set infix left 10 "⇔" ≔ eqv
+
+constant symbol select : Π (a : Type), τ a
+
+symbol ∀ {a} : (τ a → prop) → prop
+symbol ∃ {a} : (τ a → prop) → prop
+
```

Comment:
Add `set quantifier ∀` to be able to write ∀x,p instead of ∀(\\lambda x,p). Idem for \\exists.